ceph-releases/jewel/ubuntu/14.04/daemon/entrypoint.sh

```diff
@@ -246,8 +246,8 @@ function start_mon {
       exit 1
     fi
 
-    if [ ! -e /etc/ceph/monmap ]; then
-      echo "ERROR- /etc/ceph/monmap must exist.  You can extract it from your current monitor by running 'ceph mon getmap -o /etc/ceph/monmap' or use a KV Store"
+    if [ ! -e /etc/ceph/monmap-${CLUSTER} ]; then
```

Comment:
You need to change all instances of get_mon_config not just in config.static. This has broken my cluster.